src/types/docs-schemas/definitions.schema.json

```diff
@@ -373,6 +373,13 @@
         "network": { "$ref": "definitions.schema.json/#/definitions/network", "format": "uri-reference" }
       },
       "required": ["key", "network"]
+    },
+    "abi": {
+      "title": "AbiType",
+      "anyOf": [
+        { "title": "StringABI", "type": "string" },
+        { "title": "ArrayABI", "type": "array" }
```

Comment:
Should we further clarify the type of array here? e.g. `"items": {"type": ...}`

Reply:
@shahnami certainly would be useful. I tried to do that but it took me too much effort to finalize and I decided to commit in a simplified manner first. I'd propose adding this to a backlog ?